internal/server/domain/log.go

```diff
@@ -11,9 +11,14 @@ type Log struct {
 	Content            string
 }
 
+// CTRLog represents a log for click-through rate
+// eventType: the type of event e.g. click
+// createdAt: the time the event occurred e.g. "2021-01-01T00:00:00Z"
+// objectid: the id of the object that was clicked e.g. "123456"
 type CTRLog struct {
-	CreatedAt time.Time
-	Objectid  string
+	EventType string    `json:"eventType"`
```

Comment:
BTW, is JSON necessary here?
I feel like it's not needed since the domain is hidden from the presenter.

Reply:
I added it because I thought it would be easier to see, but I removed it because of the comments. I think so too.